src/components/Addform.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] `required`
- [ ] please enable the linters because it defeats the purpose of installing linters